### DISCUSSION TITLE
任务清单

### DISCUSSION THREAD

**Discussion by MKStoler4096** [ProjectCampus-CH/exam-showboard-next#7] [General]
#4 修复进度：
- [ ] 考试开始后全屏显示当前考试科目、结束倒计时，隐藏考试列表
- [ ] 考试结束前15分钟醒目提醒
- [ ] 考试结束后重新显示考试列表
- [x] 考试列表按日期分开，标题栏字体加粗
- [x] 统一当前科目、考试时间、考试状态之间的行间距
- [x] 隐藏右侧滑动条区域

**Comment by Jursin** [comment]
![Screenshots.png](https://github.com/user-attachments/assets/2263a60e-28cf-4338-b5dd-16ab5d166217)

希望考试结束文字能大一点并居中

**Reply by MKStoler4096** [reply]
好的

**Reply by MKStoler4096** [reply]
已解决 等下个版本（）
